pkg/noderesourcetopology/README.md

```diff
@@ -98,6 +98,24 @@ profiles:
       cacheResyncPeriodSeconds: 5
 ```
 
+#### ScoringStrategy
+
+Topology-aware scheduler supports four scoring strategies. You can set a strategy via SchedulerConfigConfiguration, by setting `scoringStrategy` option. There are four supported strategies:
+
+* MostAllocated
+* BalancedAllocation
+* LeastAllocated
+* LeastNUMANodes
+
+MostAllocated, BalancedAllocation and LeastAllocated strategies work only for `single-numa-node` Topology Manager policy and indicate how score of the worker node will be calculated based
```

Comment:
I'd write: "The MostAllocated, BalancedAllocation and LeastAllocated strategies only work with the `single-numa-node` Topology Manager policy"

Reply:
👍